portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,126 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Lee's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@300&family=Muli:wght@300&display=swap" rel="stylesheet">
     <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/d691fbec03.js" crossorigin="anonymous"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+    <div class="container navbar">
+      <header>
+        <div class="nav-name">
+          <a href="#"><h1>Lee</h1></a>
+        </div>
+        <nav>
+          <ul>
+            <li><a href="#about">About</a></li>
+            <li><a href="#skills">Skills</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+      </header>
     </div>
+
+    <div class="space" id="top"></div>
+
+    <div class="container main-info-section">
+      <div class="text">
+        <h1>
+          <span>H</span>
+          <span class="m-left">e</span>
+          <span class="m-left">l</span>
+          <span class="m-left">l</span>
+          <span class="m-left">o</span>
+          <span class="m-left">,</span>
+          <br>
+          <span>I</span>
+          <span class="m-left">'</span>
+          <span class="m-left">m</span>
+          <span class="m-left"> </span>
+          <span class="m-left name">L</span>
+          <span class="m-left name">e</span>
+          <span class="m-left name">e</span>
+        </h1>
+        <p>Interning at Google in the summer of 2020 <br> Part of Student Training in Engineering Program</p>
+        <a href="#about" onclick="addRandomGreeting()">begin</a>
```

Comment:
A small improvement would be if each section had a next button that took you to the next section. After clicking, begin to start, it makes me feel like there should be another button to continue.